conf/app.routes

```diff
@@ -1,4 +1,5 @@
 # microservice specific routes
 
-GET        /:id             uk.gov.hmrc.incometaxsessiondata.controllers.SessionController.getById(id: String)
-POST       /                uk.gov.hmrc.incometaxsessiondata.controllers.SessionController.set()
+GET        /:mtditid                    uk.gov.hmrc.incometaxsessiondata.controllers.SessionController.get(mtditid: String)
+GET        /get-by-mtditid/:mtditid     uk.gov.hmrc.incometaxsessiondata.controllers.SessionController.getByMtditid(mtditid: String)
```

Comment:
we don't need this endpoint atm

Reply:
i thought it could be useful for the testOnly page in the FE?

Reply:
ok, but we can't expose extra method in the service just because we need to fit this for the testing purposes;
should be easy to change agent in our test page for this reason?